.github/workflows/cross-chain-arbitrum.yml

```diff
@@ -1,5 +1,10 @@
 name: Cross chain Arbitrum
 
+# There are a couple of places where we hardcode the environment name to
+# "goerliArbitrum". As of adding this config file, there is no easy way of passing
+# an additional paramater to this workflow like ${{ github.event.inputs.environment-l2 }}
+# and would require a rebuild of the existing CI workflows across the project.
```

Comment:
> would require a rebuild of the existing CI workflows across the project.

I would rather say it's about requiring rebuild of the custom GH actions used to handle the triggering of workflows in the CI flow.